### PR TITLE
Phylogeny

### PR DESCRIPTION
**Pull request by bethecon**
New notebook for phylogeny using brapi data. Modifications to phylogeny data using flatfiles. These are based upon previous phylogeny nb already reviewed by @btmonier. @agostof Could you review these?

### CODE REVIEW

File: src/templates/01_rTASSEL_load_data.brapi.ipynb [hunk 1]
```diff
@@ -320,13 +354,13 @@
...
    "source": [
-    "PhenoDataFromGigwa_modified <- PhenoDataFromGigwa |> select(germplasmName, EH_M_cm, Q1, Q2, Q3) |>\n",
-    "    mutate(\n",
-    "        EH_M_cm = as.numeric(EH_M_cm),\n",
-    "        Q1 = as.numeric(Q1),\n",
-    "        Q2 = as.numeric(Q2),\n",
-    "        Q3 = as.numeric(Q3)\n",
-    "        )"
+    "#PhenoDataFromGigwa_modified <- PhenoDataFromGigwa |> select(germplasmName, trait1, trait2, trait3, trait4) |>\n",
+    "#    mutate(\n",
```
Comment (btmonier) [inline src/templates/01_rTASSEL_load_data.brapi.ipynb:358]: Consider using more "[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)" principles here (e.g. `all_of()`, `mutate_at()`, etc. from `dplyr`) here. For example:

```r
traitIds <- c("trait1", "trait2", "trait3", "trait4")

PhenoDataFromGigwa_modified <- PhenoDataFromGigwa |>
    select(all_of(traitIds)) |>
    mutate_at(traitIds, as.numeric)
```

File: src/templates/03_rTASSEL_SeqDiversity.brapi.ipynb [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16eeac41-692a-44d5-a161-438d97c8dd2c",
+   "metadata": {},
+   "source": [
+    "You can find more information about rTASSEL [here](https://rtassel.maizegenetics.net)\n",
+    "\n",
+    "and an rTASSEL tutorial in binder [here](https://mybinder.org/v2/gh/btmonier/rTASSEL_sandbox/HEAD?labpath=getting_started.ipynb)."
+   ]
```
Comment (btmonier) [inline src/templates/03_rTASSEL_SeqDiversity.brapi.ipynb:483]: Add Khaled's QBMS citations.

File: src/templates/05_rTASSEL_Phylogeny.brapi.ipynb [hunk 3]
```diff
@@ -0,0 +1,632 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8aa379d8-149b-439a-841a-ef0d20eba51c",
+   "metadata": {},
+   "source": [
+    "**Please also cite QBMS using the following citation:**\n",
+    "\n",
+    "Al-Shamaa K (2023). QBMS: Query the Breeding Management System(s). R package version 0.9.1, https://icarda-git.github.io/QBMS/.You can find more information about rTASSEL [here](https://rtassel.maizegenetics.net)\n",
```
Comment (btmonier) [inline src/templates/05_rTASSEL_Phylogeny.brapi.ipynb:599]: Add space after "`https://icarda-git.github.io/QBMS/.`" It's hyperlinking the "You" after it.

File: src/templates/03_rTASSEL_SeqDiversity.brapi.ipynb (hunk 2 above)
Comment (bethecon) [reply src/templates/03_rTASSEL_SeqDiversity.brapi.ipynb:483]: Added.

File: src/templates/01_rTASSEL_load_data.brapi.ipynb (hunk 1 above)
Comment (bethecon) [reply src/templates/01_rTASSEL_load_data.brapi.ipynb:358]: Changed as suggested.

File: src/templates/05_rTASSEL_Phylogeny.brapi.ipynb (hunk 3 above)
Comment (bethecon) [reply src/templates/05_rTASSEL_Phylogeny.brapi.ipynb:599]: Space added.

**Review by btmonier — DISMISSED** [review]

**Review by btmonier — APPROVED** [review]

**Review by btmonier — DISMISSED** [review]

**Comment by bethecon** [comment]
@btmonier sorry, I hadn't finished pushing the changes before you approved!

**Comment by btmonier** [comment]
Not a problem

**Review by btmonier — DISMISSED** [review]

**Review by btmonier — DISMISSED** [review]

**Review by btmonier — DISMISSED** [review]

**Review by btmonier — APPROVED** [review]

**Pull request closed without merging**